src/SubFolder.php

```diff
@@ -50,7 +50,7 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
                 throw new BadUriPrefixException('Wrong URI prefix value.');
             }
 
-            if (!str_starts_with($path, $baseUrl)) {
+            if (!empty($path) && !str_starts_with($path, $baseUrl)) {
```

Comment:
Why? What is different between "" (empty path) and "/" ?

Reply:
If prefix is set then index page doesn't work.

Hmm... maybe  it's makes sense?

Reply:
In context `SubFolder` can add expression:

```php
if ($path === '') {
  $path = '/';
}
```

Reply:
In context `SubFolder` middleware path value "empty string" and `/` is same. It's mean root directory without prefix.

Which unexpected behavior occurs?

Reply:
Application must live under prefix and shouldn't work in root path (even root is '' or '/'). In context `SubFolder`, the middleware in auto mode works like this.